config/modules_config/default-flatpaks.yml

```diff
@@ -0,0 +1,38 @@
+# The default-flatpaks module can be used to install or uninstall Flatpaks from a configurable remote on every boot. It skips that operation if no changes are detected.
+
+type: default-flatpaks # if included, https://github.com/ublue-os/yafti will be installed and set up
+notify: true # Send notification after install/uninstall is finished (true/false)
+#--------------------------------------------------------------------------------------------------#
+system:
+    # If no repo information is specified, Flathub will be used by default
+    repo-url: https://dl.flathub.org/repo/flathub.flatpakrepo
+    repo-name: flathub
+    repo-title: "Flathub (system-wide)" # Optional; this sets the remote's user-facing name in graphical frontends like GNOME Software
+    install:
+     # - Flatseal (Permission Manager): com.github.tchx84.Flatseal
```

Comment:
Format for this module is wrong.

It should be just

      # - com.github.tchx84.Flatseal
      
you can type that this is Flatseal (Permission Manager) adding additional #

Reply:
fixed